File: Back-end/src/DevIO.Business/Models/Cliente.cs
using System;
using System.Collections.Generic;

namespace DevIO.Business.Models
{
    public class Cliente : Entity
    {
        public string Nome { get; set; }
        public string Cpf { get; set; }
        public string Rg { get; set; }
        public DateTime DataNascimento { get; set; }
        public IEnumerable<Telefone> Telefones { get; set; }
        public IEnumerable<Endereco> Enderecos { get; set; }
        public IEnumerable<MidiaSocial> MidiasSociais { get; set; }

    }
}

File: Front-end/src/app/cliente/models/cliente.ts
import { Endereco } from './endereco';
import { MidiaSocial } from './midiaSocial';
import { Telefone } from './telefone';

export class Cliente {
    id: string;
    nome: string;
    cpf: string;
    rg: string;
    dataNascimento: Date;
    enderecos: Endereco[];
    telefones: Telefone[];
    midiasSociais: MidiaSocial[];
}

